guides/common/modules/proc_updating-smart-proxy.adoc

```diff
@@ -4,6 +4,10 @@
 Update {SmartProxyServers} to the next minor version.
 
 .Procedure
+ifdef::satellite[]
+. Synchronize the {RepoRHEL8ServerSatelliteCapsuleProjectVersion} repository in the {ProjectServer}.
+Publish and promote a new version of the content view with which the {SmartProxy} is registered.
```

Comment:
```suggestion
. Publish and promote a new version of the content view with which the {SmartProxy} is registered.
```
Sorry to be nit-picky, but I just realized this should be probably its own step.